### ISSUE TITLE
Cannot Add Images to Build (Workaround?)

### ISSUE THREAD

**Issue by dominicbarnes** [componentjs/builder.js#160]
I'm trying to put together a builder plugin to assemble images into a sprite during the build, but can't seem to inject images, only styles.

Using component 0.19 (w/ builder 0.10) `Builder#addFile("images", ...)` doesn't behave like I would expect. It turns out that static assets like images/fonts/etc don't have a hook, and are fired off right away, so the copy/link operation is likely done (or at least in-flight) before I can get to it.

Is this changed in the later versions of the builder? (v0.11+) Is there a recommended workaround right now?

**Comment by ianstormtaylor** [comment]
yeah this would be accessible in the newest version of the builder since you could choose not to copy them over and to instead sprite em

**Comment by dominicbarnes** [comment]
In my particular case, I'm using a separate `sprite` config, rather than `images`. (although either could be reasonable)

Should I just make a custom build script using the newest builder.js? (are there any incompatibilities or breaking changes I may need to be aware of?)

**Comment by ianstormtaylor** [comment]
yeah `sprites` makes sense to be able to know which are not to be sprited. builder plugin would be good, although i'm currently waiting on https://github.com/component/builder.js/pull/151 before i get too into using the new api cuz we gotta solve the verbosity stuff
